draft-ietf-quic-transport.md

```diff
@@ -2860,12 +2860,13 @@ valid frames? -->
 ### Sending ACK Frames
 
 An endpoint MUST NOT send more than one packet containing only an ACK frame per
```

Comment:
FWIW, the point of my previous comment https://github.com/quicwg/base-drafts/pull/2794#discussion_r295118147 was that "MUST NOT send more than one packet containing only an ACK frame" does not prohibit one from sending a packet that contains ACK and PADDING.

This is admittedly a nit-pick, but discussing ACK-only on the send side while discussing ACK+PADDING on the receive side is confusing.

Hence the suggestion to change the text to "MUST NOT send more than one non-ACK-eliciting packet per ..."